modules/features/referrals/src/main/kotlin/au/com/shiftyjelly/pocketcasts/referrals/ReferralsViewModel.kt

```diff
@@ -31,15 +31,17 @@ class ReferralsViewModel @Inject constructor(
                 userManager.getSignInState().asFlow(),
                 settings.playerOrUpNextBottomSheetState,
             ) { signInState, playerBottomSheetState ->
-                val showIcon = FeatureFlag.isEnabled(Feature.REFERRALS) && signInState.isSignedInAsPlusOrPatron
+                val eligibleToSendPass = FeatureFlag.isEnabled(Feature.REFERRALS) && signInState.isSignedInAsPlusOrPatron
+                val eligibleToClaimPass = FeatureFlag.isEnabled(Feature.REFERRALS) // TODO: Fix condition to claim pass when it's implemented
                 _state.update {
                     it.copy(
-                        showIcon = showIcon,
+                        showIcon = eligibleToSendPass,
                         showTooltip = if (playerBottomSheetState == BottomSheetBehavior.STATE_COLLAPSED) {
-                            showIcon && settings.showReferralsTooltip.value
+                            eligibleToSendPass && settings.showReferralsTooltip.value
                         } else {
                             false
                         },
+                        showProfileBanner = eligibleToClaimPass,
```

Comment:
I haven't added tests for claim pass profile banner visibility as the eligibility to claim pass will be changed later.